Lib/test/test_os.py

```diff
@@ -2560,6 +2560,12 @@ def test_waitpid(self):
         status = os.waitpid(pid, 0)
         self.assertEqual(status, (pid, 0))
 
+class AffinityTests(unittest.TestCase):
```

Comment:
Can you put this in `test_posix` with the other `setaffinity` tests?

Reply:
Thanks for pointing that out... I hadn't noticed `test_posix.py`!